flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/functions/SystemFunctionUtils.java

```diff
@@ -184,6 +231,53 @@ public static int timestampDiff(String symbol, long fromDate, long toDate) {
         }
     }
 
+    public static TimestampData timestampadd(
+            String timeintervalunit, int interval, LocalZonedTimestampData timepoint) {
+        return timestampadd(timeintervalunit, interval, timepoint.getEpochMillisecond());
+    }
+
+    public static TimestampData timestampadd(
+            String timeintervalunit, int interval, ZonedTimestampData timepoint) {
+        return timestampadd(timeintervalunit, interval, timepoint.getMillisecond());
+    }
+
+    public static TimestampData timestampadd(
+            String timeintervalunit, int interval, TimestampData timepoint) {
+        return timestampadd(timeintervalunit, interval, timepoint.getMillisecond());
+    }
+
+    public static TimestampData timestampadd(
+            String timeintervalunit, int interval, long timepoint) {
```

Comment:
Use camel-case names for parameters, like `timeIntervalUnit`. Please also check other newly-added functions' parameters.